express-post-json/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+const middleware = express.json();
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
+
+let nextId = 1;
+const grades = {};
+
+app.use(middleware, function (req, res, next) {
+  next();
+});
+
+const arr = [];
+app.get('/api/grades', function (req, res) {
+  for (const keys in grades) {
+    arr.push(grades[keys]);
+  }
+  res.json(arr);
+});
```

Comment:
Careful, if you send this server a GET request multiple times in a row after POSTing something, the array will keep growing in length (see for yourself). Try moving your array declaration inside of the `app.get()` callback.